bundles/org.openhab.binding.siemensrds/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,189 @@
+			</channel>
+
+			<channel id="roomAirQuality" typeId="airQuality">
+		   		<label>Room Air Quality</label>
+		   		<description>Actual room air quality</description>
+			</channel>
+
+			<channel id="outsideTemperature" typeId="temperature">		   		
+		   		<label>Outside Temperature</label>
+		   		<description>Actual outside temperature</description>
+			</channel>
+
+			<channel id="greenLeaf" typeId="greenLeaf">
+		   		<label>Green Leaf Mode</label>
+   				<description>Energy saving level</description>
+			</channel>
+
+			<channel id="statProgramMode" typeId="statProgramMode">
+				<label>T&apos;stat Program (Manual::Auto)</label>
+		   		<description>The program mode of the thermostat (user can override)</description>
+			</channel>
+
+			<channel id="statOccupancyState" typeId="statOccupancyState">
+		   		<label>Occupancy State (Absent::Present)</label>
+   				<description>The occupancy mode of the thermostat (user can override)</description>
+			</channel>
+
+			<channel id="dhwProgramMode" typeId="dhwProgramMode">
+				<label>DHW Program (Manual::Auto)</label>
+		   		<description>The program mode of the domestic water heating (user can override)</description>
+			</channel>
+			
+			<channel id="dhwOffOnState" typeId="dhwOffOnState">
+				<label>DHW Switch State (Off::On)</label>
+		   		<description>The state of the domestic water heating (user can override)</description>
+			</channel>
+
+		</channels>
+
+		<properties>
+			<property name="vendor">Siemens</property>
+			<property name="modelId">RDS</property>
+			<property name="climatixIcPlantId"></property>
+        	</properties>
+        <representation-property>climatixIcPlantId</representation-property>
+
+	</thing-type>
+
+	<channel-type id="temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<description>Measured temperature value</description>
+   		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="setTemperature">
+   		<item-type>Number:Temperature</item-type>
+   		<label>Set Temperature</label>
+		<description>Target temperature setting</description>
+   		<state readOnly="false" pattern="%.1f %unit%" min="5" max="30" step="0.5"/>
+	</channel-type>
+	
+	<channel-type id="heatingOffCooling">
+		<item-type>String</item-type>
+		<label>Heating / Cooling</label>
+   		<description>Heating / Cooling state</description>
+   		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="humidity">
+		<item-type>Number</item-type>
+		<label>Humidity</label>
+		<description>Measured humidity value</description>
+   		<state readOnly="true" pattern="%.0f %%r.H."/>
+	</channel-type>
+
+	<channel-type id="airQuality">
+		<item-type>String</item-type>
+		<label>Air Quality</label>
+		<description>Room Air Quality</description>
+   		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="greenLeaf">
+		<item-type>String</item-type>
+		<label>Energy savings level </label>
```

Comment:
Please use camel case for labels (`Energy Savings Level`)

Reply:
Ok.